app/src/main/java/com/example/criminalintent/presentation/crime_detail/CrimeDetailViewModel.kt

```diff
@@ -18,13 +18,12 @@ class CrimeDetailViewModel : ViewModel() {
     val checkboxClickListener = View.OnClickListener {
         crime.isSolved = (it as CheckBox).isChecked
     }
-
     var crimeLiveData: LiveData<Crime?> =
         Transformations.switchMap(crimeIdLiveData) { crimeId ->
             crimeRepository.getCrime(crimeId)
         }
 
-    fun updateSuspect(suspect: String) {
+    fun onSuspectNameSelected(suspect: String) {
```

Comment:
As said in a previous comment, the name of this param is wrong. It is in fact the suspectName